README.md

```diff
@@ -1 +1,2 @@
-# Build Scraper
+# Basic Web Scraper
+A scraper that scraps the data of King County Food Inspection.
```

Comment:
while accurate, this description is a bit thin.  What else might a visitor to this repository want to know about it before needing to look at the code?  What specifically does it scrape? (note the misspelling in your readme).  What does it use?  To what purpose might this data be put?  How might one extend the data with additional information if needed?